framed.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var Client = require('ifrau/client/slim'),
+var Client = require('./ifrau-client'),
```

Comment:
I'm a little surprised this works... returning a promise from a CJS import, but if it works -- sweet!

Reply:
It certainly seemed like it did, but I'm planning to run through some more detailed test cases to make absolutely sure it's not just some phantom thing (like a promise not being awaited) making it look like it's functional. Worst case scenario I can dump this back in the other file; it just means we can't isolate the stuff we're not linting as well. Unfortunate, but not a deal-breaker imo.

Reply:
Ah. Well. I made that assumption on the grounds that the tests passed. But the test file for this class is empty. So about all I can say is that `require('./framed')` doesn't throw 😄. I'll dig deeper.

Reply:
So this did need a tweak. Specifically, `require`ing a promise worked fine, but in a bit of silliness I wasn't actually waiting for that promise to resolve before chaining into the next promise. So now the client loading is the first chained promise.